lib/ReactViews/BottomDock/MapDataCount.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { observer } from "mobx-react";
+
+import Icon, { StyledIcon } from "../Icon";
+import ButtonAsLabel from "../../Styled/ButtonAsLabel";
+import Box from "../../Styled/Box";
+import Text from "../../Styled/Text";
+import Spacing from "../../Styled/Spacing";
+import { withTranslation } from "react-i18next";
+
+const MapDataCount = withTranslation()(
+  observer(function(props) {
+    const { t, terria, viewState } = props;
+    if (viewState.useSmallScreenInterface) {
+      return null;
+    }
+
+    // Can't simply use number of items given they can exist in workbench
+    // without being shown on map
+    const numberOfDatasets = terria.workbench.items.filter(item => item.show)
+      .length;
+    const hasMapData = numberOfDatasets !== 0;
+    const dataset = t("countDatasets.datasetSingular");
+    const datasetPlural = t("countDatasets.datasetPlural");
+    const datasetString =
+      numberOfDatasets > 1 || numberOfDatasets === 0 ? datasetPlural : dataset;
+    const mapDataText = hasMapData
```

Comment:
Would it be more appropriate to rewrite this in the i18next interpolation style?

Reply:
Functionally the same but there could be the case where the grammar/order works differently for a different language so I've moved a tiny bit of the logic into the translation file